lib/pure/net.nim

```diff
@@ -247,7 +247,7 @@ proc newSocket*(domain: Domain = AF_INET, sockType: SockType = SOCK_STREAM,
     raiseOSError(osLastError())
   result = newSocket(fd, domain, sockType, protocol, buffered)
 
-proc parseIPv4Address(addressStr: string): IpAddress =
+proc parseIPv4Address*(addressStr: string): IpAddress =
   ## Parses IPv4 addresses
   ## Raises ValueError on errors
```

Comment:
You should at least fix how this is rendered (check via `nim doc net.nim`). You'll see that it's just one sentence which is wrong, you need to add an empty ``##`` between them to ensure the line break is there in the docs.

But really, ideally, we would like runnable examples here too.